traceapp/app.go

```diff
@@ -39,6 +39,8 @@ type App struct {
 
 	tmplLock sync.Mutex
 	tmpls    map[string]*htmpl.Template
+
+	log bool
```

Comment:
Shouldn't this be exported since you have no other way of setting this? I assume there would be cases where you'd want to disable the extra logging.